lib/inventory/inventory_api.rb

```diff
@@ -57,6 +57,23 @@ def list_resource_types(feed = nil)
       ret.map { |rt| ResourceType.new(rt) }
     end
 
+    # Return all resources for a feed
+    # @param [String] feed Id of the feed that hosts the resources
+    # @param [Boolean] fetch_properties Should the config data be fetched too
+    # @return [Array<Resource>] List of resources, which can be empty.
+    def list_resources_for_feed(feed, fetch_properties = false)
+      fail 'Feed must be given' unless feed
+      the_feed = hawk_escape(feed)
```

Comment:
In some places, there are parenthesis when calling hawk_escape, in other places it's without.

Also, wouldn't it be a good candidate for a monkey patching, specially if this escape is used in other non-inventory places? No need to make it system wide, though.